app.js

```diff
@@ -1,54 +1,76 @@
 var hours = [
-  '10am: ',
-  '11am: ',
-  '12pm: ',
-  '1pm: ',
-  '2pm: ',
-  '3pm: ',
-  '4pm: ',
-  '5pm: '
+  '10am',
+  '11am',
+  '12pm',
+  '1pm',
+  '2pm',
+  '3pm',
+  '4pm',
+  '5pm'
 ];
+var stores = [];
+var tableView;
 
-Store.prototype.totals = [];
-Store.prototype.grandTotal = 0;
+// Store.prototype.totals = [];
+// Store.prototype.grandTotal = 0;
 
 function Store (storeName, minCust, maxCust, avgCookie, id) {
   this.storeName = storeName;
```

Comment:
We had to add that id....i don't remember why but it was something you suggested....so in my instances i'm adding in - after the 5.2 - the id...  However right now I honestly can't remember why that was added.